domains/bug.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "gotoobe"
+    },
+    "records": {
+        "CNAME": "bug-is-a-dev.pages.dev"
+    },
+    "proxied": true
```

you cant proxy cf pages
```suggestion
    }
```